src/languages/twig.js

```diff
@@ -27,17 +27,17 @@ function(hljs) {
   var FILTER = {
     begin: /\|[A-Za-z_]+:?/,
     keywords:
-      'abs batch capitalize convert_encoding date date_modify default ' +
-      'escape first format join json_encode keys last length lower ' +
-      'merge nl2br number_format raw replace reverse round slice sort split ' +
-      'striptags title trim upper url_encode',
+      'abs batch capitalize column convert_encoding date date_modify default ' +
+      'escape filter first format inky inline_css join json_encode keys last ' +
+      'length lower map markdown merge nl2br number_format raw reduce replace ' +
+      'reverse round slice sort spaceless split striptags title trim upper url_encode',
     contains: [
       FUNCTIONS
     ]
   };
 
-  var TAGS = 'autoescape block do embed extends filter flush for ' +
```

Comment:
The failing test is either being caused by the removal of filter here or the addition above. In the interest of getting the PR merged, try re-adding it here or removing it above. Afterall, filter IS still a valid tag, just deprecated.

Reply:
Oh yes is it’s only deprecated I would leave it for sure. That would be an easy fix.